### PR TITLE
Release v6.7.6 lacework-agent Helm Charts

### PR DESCRIPTION
**Pull request by rnalexlacework**
This is a helm chart update for 6.7.6.

### CODE REVIEW

**Review by anilnanduri — APPROVED** [review]
Looks good to me, as we talked,  the only files that we checkin are index.yaml and tgz file for 6.7.6

File: lacework-agent/Chart.yaml [hunk 1]
```diff
@@ -14,4 +14,4 @@ maintainers:
 - email: info@lacework.net
   name: lacework-support
 name: lacework-agent
-version: 6.9.0
+version: 6.7.6
```
Comment (nschmeller) [inline lacework-agent/Chart.yaml:17]: Are we intentionally modifying this value? My understanding is that we're publishing the artifacts but not changing the default version of the chart

File: lacework-agent/templates/configmap.yaml [hunk 2]
```diff
@@ -36,13 +36,6 @@ data:
        },
        {{- end }}
      {{- end }}
-     {{- with .Values.laceworkConfig.codeaware }}
```
Comment (nschmeller) [inline lacework-agent/templates/configmap.yaml:39]: Same here, we shouldn't remove this

**Review by nschmeller — APPROVED** [review]
+1 to @anilnanduri's comment

**Pull request merged**